util.go

```diff
@@ -108,6 +109,31 @@ func utfrune(s []rune, r rune) int {
 	return -1
 }
 
+func cvttorunes(p []byte, n int) (r []rune, nb int, nulls bool) {
```

Comment:
Would you mind terribly adding godoc here in a future CL? I know that most of the other code doesn't have godoc but I am trying to (slowly) add it when I add new code.